lib/jazzy/assets/css/jazzy.css.scss

```diff
@@ -1,925 +1,416 @@
+////////////////////////////////
+// Constants
+////////////////////////////////
+
+$bg_color: #414141;
 $doc_coverage_color: #999;
+$code_color: #777;
+$code_bg_color: #eee;
+$link_color: #0088cc;
+$white_color: #fff;
+$light_gray_bg_color: #f2f2f2;
+$declaration_bg_color: #f9f9f9;
+$sidebar_bg_color: #f9f9f9;
+$declaration_lang_title_swift_color: #4b8afb;
+
+$sidebar_width: 230px;
 $content_wrapper_width: 980px;
-$code-font: 12px Menlo, monospace;
-$graybox-border: 1px solid rgba(233, 233, 233, 1);
+$content_top_offset: 70px;
+$content_body_margin: 16px;
+$content_body_left_offset: $sidebar_width + $content_body_margin;
+$header_height: 26px;
+$breadcrumb_padding_top: 17px;
+
+$code_font: 0.95em Menlo, monospace;
+
+$gray_border: 1px solid #e2e2e2;
+$declaration_swift_border: 5px solid #cde9f4;
```

Comment:
it is Swift specific. We'll use a different color for Objective-C declarations when the time comes. Same reason I have `div.Swift` further down.